pytest.ini

```diff
@@ -26,7 +26,7 @@
 pep8ignore =
     docs/conf.py ALL
     config.py E501
-addopts = --pep8 --doctest-glob="*.rst" --doctest-modules --cov=rero_ils --cov-report=term-missing --ignore=setup.py
+addopts = --pep8 --doctest-glob="*.rst" --doctest-modules --cov=rero_ils --cov-report=term-missing --ignore=setup.py -m "not externaltest"
```

Comment:
instead of ignoring tests you can use a specific pytest options such as: http://doc.pytest.org/en/latest/example/simple.html#control-skipping-of-tests-according-to-command-line-option

Reply:
as discussed, it is better to use the ignore option